core/app/(user)/magic-link/route.ts

```diff
@@ -85,7 +85,7 @@ export async function GET(req: NextRequest) {
 
 	const validatedTokenPromise = validateToken(token);
 
-	const currentSessionCookie = cookies().get(lucia.sessionCookieName)?.value;
+	const currentSessionCookie = (await cookies()).get(lucia.sessionCookieName)?.value;
```

Comment:
most of the changes are of this caliber: `await`ing `cookies`, `headers`, `params`, and `searchParams`. I will comment on more nuanced changes